### PR TITLE
More cmdline processing fixes

### PR DESCRIPTION
**Pull request by clumens**
However, note that this is not perfect.  See the commit message in e9d87e6 for details.

### CODE REVIEW

**Comment by clumens** [comment]
This could probably be extended a bit to support the option taking the argument being last in the list.  That is, `-abc "-1i3"` could be turned into `-a -b -c "-1i3"`.

**Comment by clumens** [comment]
In fact, doing so makes for a much shorter patch.  Updated.

File: tools/crm_attribute.c [hunk 1]
```diff
@@ -312,7 +312,7 @@ main(int argc, char **argv)
 
     GOptionGroup *output_group = NULL;
     pcmk__common_args_t *args = pcmk__new_common_args(SUMMARY);
-    gchar **processed_args = pcmk__cmdline_preproc(argv, "DGNPdilnpstv");
+    gchar **processed_args = pcmk__cmdline_preproc(argv, "NPdilnpstv");
```
Comment (kgaillot) [inline tools/crm_attribute.c:315]: also U

File: tools/crmadmin.c [hunk 2]
```diff
@@ -188,7 +188,7 @@ main(int argc, char **argv)
 
     GOptionGroup *output_group = NULL;
     pcmk__common_args_t *args = pcmk__new_common_args(SUMMARY);
-    gchar **processed_args = pcmk__cmdline_preproc(argv, "itBDEHKNPS");
+    gchar **processed_args = pcmk__cmdline_preproc(argv, "itHKNS");
```
Comment (kgaillot) [inline tools/crmadmin.c:191]: not H

File: tools/crm_attribute.c (hunk 1 above)
Comment (clumens) [reply tools/crm_attribute.c]: I checked these two or three times, and somehow still missed this and the other one.  Fixed both.

**Pull request merged**